docs/guides/tutorials/rest_apis_with_fastapi.rst

```diff
@@ -29,10 +29,10 @@ Prerequisites
 =============
 
 Before we start, make sure you've :ref:`installed <ref_admin_install>` the
-``edgedb`` command line tool. In this tutorial, we'll use Python 3.10 and take
-advantage of the asynchronous I/O paradigm to communicate with the database
-more efficiently. If you want to skip ahead, the completed source code for this
-API can be found `in our examples repo
+``edgedb`` command line tool. For this tutorial, we'll need Python 3.4 or 
+newer to take advantage of the asynchronous I/O paradigm to communicate with 
+the database more efficiently. If you want to skip ahead, the completed 
```

Comment:
My motivation here was that `asyncio` was introduced in 3.4 (please correct me if I'm wrong) and there doesn't seem to be any other requirements for >3.10

Reply:
I don't have a problem with it in principle. Were you able to test the guide on 3.4? I was going to try, but apparently it's challenging to install anything prior to 3.7 on a Mac with Apple Silicon.

Reply:
I decided to revert this change, but added a sentence that you can use a newer version as well.